Readme.md

```diff
@@ -32,89 +32,139 @@ const cryptoFriends = setup(friends) // sodium-universal variant of crypto
 const cryptoSodium = setup(sodium) // libsodium.js variant of crypto
 ```
 
-## Sending/Receiving encrypted messages
+## Vocabulary
+
+- `Channel` - an e2e encrypted setup consisting of one `Receiver` and one `Sender`
+- `Sender` - an object containing the keys that allow to encrypt data
+- `Receiver` - an object containing the keys that allow to decrypt data created by the `Sender`
+- `Connection` - an e2e encrypted setup consisting of the `Receiver` of one channel and the `Sender` of another.
+- `Annonymous` - an object describing an the capability to verify if a message is part of a `Channel`
```

Comment:
Would Verifier maybe be better wording for this? The name isn't super intuitive at the moment. Else maybe a description of why Anonymous was used could be helpful. (is it a crypto term?)

Reply:
I think that's fixed in #7 😉